samples/wifi/ble_coex/README.rst

```diff
@@ -18,50 +17,49 @@ The sample supports the following development kit:
 
 .. table-from-sample-yaml::
 
+Running coexistence sample test cases require additional software such as the Wi-Fi **iperf** application.
+Use **iperf** version 2.0.5.
```

Comment:
Using **iPerf** to emphasize the first time is ok. But I would suggest not to use it throughout the doc. To just use iPerf should be enough.

Reply:
To add to what @mia-ko wrote: bold is only accepted for PCB and GUI elements in the docs. https://nordicsemi.atlassian.net/wiki/spaces/TECHDOC/pages/120291747/Typographical+conventions